suites/apis/centralizedAuth.js

```diff
@@ -0,0 +1,639 @@
+     make sure they still can't create, read, update OR delete in /gen3
+*/
+Scenario('User with access can CRUD indexd records in namespace, not outside namespace @indexdJWT',
+  async (fence, indexd, users, files) => {
+    // create
+    const gen3_create_success = await indexd.do.addFileIndices(
+        Object.values(new_gen3_records), users.mainAcct.accessTokenHeader);
+    const abc_create_success = await indexd.do.addFileIndices(
+        Object.values(new_abc_records), users.mainAcct.accessTokenHeader);
+
+    // asserts for create
+    chai.expect(
+      gen3_create_success,
+      'uh oh, user without permission was able to register files to indexd under `/gen3`'
+    ).to.be.false;
+    chai.expect(
+      abc_create_success,
+      'user with permission was NOT able to register files to indexd under `/abc`'
+    ).to.be.true;
+
+    // read
+    const gen3_read_response = indexd.do.getFile(
+        new_gen3_records.fooBarFile, users.mainAcct.accessTokenHeader)
+    const abc_read_response = indexd.do.getFile(
+        new_abc_records.fooBarFile, users.mainAcct.accessTokenHeader)
+
+    // asserts for read
+    fenceQuestions.assertStatusCode(
+        gen3_read_response, 403,
+        msg='should have gotten unauthorized for reading record under `/gen3`'
+    );
+    fenceQuestions.assertStatusCode(
+        abc_read_response, 200,
+        msg='user with permission could not read record under `/abc`'
+    );
+
+    // update
+    const filename_change = {
+      'file_name': 'test_filename'
+    }
+    const gen3_update_response = indexd.do.updateFile(
+        new_gen3_records.fooBarFile, filename_change, users.mainAcct.accessTokenHeader)
+    const abc_update_response = indexd.do.updateFile(
+        new_abc_records.fooBarFile, filename_change, users.mainAcct.accessTokenHeader)
+
+    // asserts for update
+    fenceQuestions.assertStatusCode(
+        gen3_update_response, 403,
+        msg='should have gotten unauthorized for updating record under `/gen3`'
+    );
+    fenceQuestions.assertStatusCode(
+        abc_update_response, 200,
+        msg='user with permission could not update record under `/abc`'
+    );
+    chai.expect(
+        abc_read_response, 'update response does not contain file_name'
+    ).to.have.property('file_name');
+    chai.expect(
+        abc_read_response.file_name,
+        'update record response does not have filename we updated it with'
+    ).to.have.string('test_filename');
+
+    // delete
+    const gen3_delete_response = indexd.do.deleteFile(
+        new_gen3_records.deleteMe, users.mainAcct.accessTokenHeader)
+    const abc_delete_response = indexd.do.deleteFile(
+        new_abc_records.deleteMe, users.mainAcct.accessTokenHeader)
+
+    // asserts for delete
+    fenceQuestions.assertStatusCode(
+        gen3_delete_response, 403,
+        msg='should have gotten unauthorized for deleting record under `/gen3`'
+    );
+    fenceQuestions.assertStatusCode(
```

Comment:
there is a `deleteFileSuccess` function in `indexdQuestions` that you could use instead of checking code=200. I think adding the code=200 check to that function would be nice, and if you want to be thorough you can also call `getFile` and check that no file is found (also in the `deleteFileSuccess` function)